### PR TITLE
add getting products for sale without looking for the quantity

### PR DESCRIPTION
**Pull request by MirkaaStefanov**

### CODE REVIEW

**Comment by MirkaaStefanov** [comment]
I am not sure who should be able to see this page

File: src/main/java/com/example/ludogoriesoft/lukeriaerpapi/repository/ProductRepository.java [hunk 1]
```diff
@@ -16,4 +16,8 @@ public interface ProductRepository extends JpaRepository<Product, Long> {
     Optional<Product> findByIdAndDeletedFalse(Long id);
 
     Optional<Product> findByPackageIdAndDeletedFalse(Package packageEntity);
+
+    @Query("SELECT p FROM Product p WHERE p.forSale=true")
```
Comment (GeorgiStoyanovAtanasov) [inline src/main/java/com/example/ludogoriesoft/lukeriaerpapi/repository/ProductRepository.java:20]: I don't think we need to use a query here, we can just write  List<Product> findByForSale(true); or List<Product> findByForSaleTrue();

Comment (MirkaaStefanov) [reply src/main/java/com/example/ludogoriesoft/lukeriaerpapi/repository/ProductRepository.java:20]: Does it really matter

**Comment by berki0** [comment]
> I am not sure who should be able to see this page

this page can be seen by all roles

**Review by berki0 — APPROVED** [review]

**Pull request merged**